tests/utils/normalizeSnapshot.ts

```diff
@@ -6,7 +6,10 @@ export function normalizeSnapshot(html: string) {
       // Vue renders extra whitespace inside option item elements
       .replace(/>\s+?(\w+?)\s+?<\/option>/gs, '>$1</option>')
       // Vue renders extra whitespace inside button elements
-      .replace(/>\s+(.+?)\s+<\/button>/gs, '>$1</button>')
+      .replace(
+        /(<button[^>]*>)\s+(.+?)\s+(<\/button>)/gs,
+        (_, open, content, close) => `${open}${content.trim()}${close}`
+      )
```

Comment:
The previous expression was too greedy and would sometime target non-matching button tags, this one only works in the scope of a single button element.